### PR TITLE
[Feature] introduced a new image, called LDH (Linkis's hadoop all-in-one image)

### PR DESCRIPTION
**Pull request by AaronLinOops**
### What is the purpose of the change
introduced a new image, called LDH (Linkis's hadoop all-in-one image), which provides a pseudo-distributed hadoop cluster for testing quickly.

### Brief change log
- a Dockerfile for LDH image
- some script to bring up ldh cluster easily

### Verifying this change
(Please pick either of the following options)  
This change is a trivial rework / code cleanup without any test coverage.  

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): (no)
- Anything that affects deployment: (no)
- The MGS(Microservice Governance Services), i.e., Spring Cloud Gateway, OpenFeign, Eureka.: (no)

### Documentation
- Does this pull request introduce a new feature? (yes)
- If yes, how is the feature documented? (docs)

### CODE REVIEW

File: linkis-dist/docker/ldh.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+ARG IMAGE_BASE=centos:7
+
+FROM ${IMAGE_BASE} as linkis-ldh
+
+ARG JDK_VERSION=1.8.0-openjdk
+ARG JDK_BUILD_REVISION=1.8.0.332.b09-1.el7_9
+ARG MYSQL_JDBC_VERSION=5.1.49
+
+ARG HADOOP_VERSION=2.7.0
```
Comment (peacewong) [inline linkis-dist/docker/ldh.Dockerfile:30]: It is best to match the version numbers of hive, hadoop, and spark used by linkis by default

**Review by peacewong — APPROVED** [review]

**Review by jacktao007 — CHANGES_REQUESTED** [review]
login-pod.sh  and remote-debug-proxy is empty file.

File: linkis-dist/helm/scripts/resources/ldh/configmaps/configmap-flink.yaml [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    #historyserver.web.address: 0.0.0.0
+
+    # The port under which the web-based HistoryServer listens.
+    #historyserver.web.port: 8082
+
+    # Comma separated list of directories to monitor for completed jobs.
+    #historyserver.archive.fs.dir: hdfs:///completed-jobs/
+
+    # Interval in milliseconds for refreshing the monitored directories.
+    #historyserver.archive.fs.refresh-interval: 10000
```
Comment (legendtkl) [inline linkis-dist/helm/scripts/resources/ldh/configmaps/configmap-flink.yaml:273]: empty line is recommanded

**Review by legendtkl — COMMENTED** [review]
LGTM with minor comment

**Comment by legendtkl** [comment]
LGTM

**Pull request merged**